magento/inventory
# MSI-765: RSS feed works incorrect

Seems like RSS feed functionality is broken for main M2 repo.
Internal ticket https://jira.corp.magento.com/browse/MAGETWO-89645 created.

### Test scenario:
1. Enable RSS feed in M2 configuration: Stores -> Configuration -> Catalog -> RSS feed.
2. Subscribe to http://base_url/admin/rss/feed/index/type/notifystock.
3. Try to get low stock items.
4. You will get nothing due to bug in M2(MAGETWO-89645). Try to get items again.
5. Items are returned from cache.

Notice: You need to clean cache every time you want to get different result due to configuration, bacause results are cached.

### Fixed Issues (if relevant)

1. https://github.com/magento-engcom/msi/issues/765
